src/Engines/BaseEngine.cs

```diff
@@ -39,19 +39,17 @@ public ExecutionChannel(BaseEngine engine, Message parent)
 
             public void Display(object displayable)
             {
-                if (displayable == null) throw new ArgumentNullException(nameof(displayable));
                 engine.WriteDisplayData(parent, displayable);
             }
 
             public void Stderr(string message)
             {
-                if (message == null) throw new ArgumentNullException(nameof(message));
+                if (message == null) return;
                 engine.WriteToStream(parent, StreamName.StandardError, message);
             }
 
             public void Stdout(string message)
             {
```

Comment:
Do we need a similar `if (message == null) return;` line like at line 47?

Reply:
I actually did the opposite. I let it through even if message is null.

Reply:
Makes sense, mainly wanted to suggest that both `Stdout` and `Stderr` worked in the same way.